### ISSUE TITLE
JSON Blob

### ISSUE THREAD

**Issue by divijbindlish** [sdslabs/bot#8]
@captn3m0: Any specific reason for not suing redis and going with JSON Blob?

**Comment by captn3m0** [comment]
Yes, redis instances in free heroku tier have one major limitation: They get cleaned up after reboots.
